DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -421,11 +421,19 @@ def verify_Syslog_daemon_listening(self):
         command_to_run = "sudo netstat -lnpv | grep " + self.SYSLOG_DAEMON
         result_keywords_array = [self.SYSLOG_DAEMON, "LISTEN", ":514 "]
         command_object = BasicCommand(command_name, command_to_run, result_keywords_array)
-        command_object.run_full_test()
-        if command_object.is_successful == "Warn":
-            command_object.print_warning(command_object.command_result_err)
-        elif not command_object.is_successful:
-            command_object.print_warning(self.Syslog_daemon_not_listening_warning)
+        command_object.run_command()
+        is_command_successful = command_object.is_command_successful()
```

Comment:
The function is_command_successul() already changes the command_object property called successful so there is no need to create another parameter.